pkg/apis/kops/networking.go

```diff
@@ -248,12 +248,20 @@ type RomanaNetworkingSpec struct {
 
 // AmazonVPCNetworkingSpec declares that we want Amazon VPC CNI networking
 type AmazonVPCNetworkingSpec struct {
-	// The container image name to use
-	ImageName string `json:"imageName,omitempty"`
-	// The init container image name to use
-	InitImageName string `json:"initImageName,omitempty"`
+	// AwsVpcK8sCniConfigureRpFilter specifies whether ipamd should configure rp filter for primary interface.
+	// Setting this to false will require rp filter to be configured through init container.
+	AwsVpcK8sCniConfigureRpFilter string `json:"awsVpcK8sCniConfigureRpFilter,omitempty"`
```

Comment:
All of this refers to the AWS VPC CNI, so that part of the name is redundant.

"RP" is an acronym for "Reverse Path".

The possible values are "true" and "false", so it's a boolean.

```suggestion
	ConfigureRPFilter *bool `json:"configureRPFilter,omitempty"`
```